packages/patternfly-react/src/components/ToastNotification/ToastNotificationList.js

```diff
@@ -35,7 +35,7 @@ class ToastNotificationList extends React.Component {
   renderChildren() {
     const { paused } = this.state;
     return React.Children.map(this.props.children, child => {
-      if (child && child.type.displayName === TimedToastNotification.displayName) {
+      if (child && child.type && child.type.displayName === TimedToastNotification.displayName) {
```

Comment:
This type check logic is used in a lot of places, maybe it could be broken out into a function somewhere?

Reply:
Added a helper method and used the existing `filterChildren` method.